### ISSUE TITLE
ES6 import error

### ISSUE THREAD

**Issue by miso440** [zeh/prando#2]
A TS file such as:

```ts
import Prando from 'prando';

const test = new Prando();
```

Compiled to JS and ran throws:

```log
const test = new prando_1.default();
             ^

TypeError: prando_1.default is not a constructor
    at Object.<anonymous> (/.../dist/prando.test.js:5:14)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)
    at startup (internal/bootstrap/node.js:285:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:739:3)
```

Doing the `require` method allows construction of a `Prando` object but breaks all the typing.

Same result in Node versions `8.9.2` and `10.12.0`

**Comment by zeh** [comment]
Sorry I missed this earlier! Fixing it this week.

**Comment by jcowman2** [comment]
I'm running into the same issue. When I try `import * as Prando from 'prando';`, I get the following TypeScript error: 

`Cannot use 'new' with an expression whose type lacks a call or construct signature.`.

**Comment by zeh** [comment]
@jcowman2 @miso440 Do you have a link to any project that causes this error when attempting to import Prando? The reason I'm asking is because I've been unable to replicate it; my own projects that use the package still work with the latest version. I'm thinking this is somehow related to the tsconfig settings, but haven't been able to pinpoint what yet.

**Comment by jcowman2** [comment]
@zeh Thanks for looking into this. [Here's the branch](https://github.com/regal/regal/blob/feat/52-random/src/random/func/generate-seed.ts) of my project that's failing.

**Comment by zeh** [comment]
@jcowman2 Thanks man, I'm taking a look.

**Comment by zeh** [comment]
Identified the issue, more or less. It's when using TSC directly or, in your case, ts-node. They try resolving to the `main` file of package.json (the UMD export) rather than the `module` file, which would be more kosher. Then it breaks. The funny thing is, importing the UMD file through pure JavaScript works fine.

I haven't noticed that because my standard builds use Webpack or similar packagers, and it works fine there.

Still investigating, as it's likely we just need to export the UMD file in a way that is better understood by tsc.

**Comment by zeh** [comment]
@jcowman2 I think I've fixed it. I've published a new release, under version 5.0.0. It _shouldn't_ be breaking anything (old exports are still the same), but since this is somewhat of a big impact change, it's a new major version.

Trying your repo with

```shell
npm install prando@latest --save
npm run test
```

Works well with the tests you had that were previously broken.

Can you try again?

**Comment by jcowman2** [comment]
I've just confirmed, it works in my project now. Thanks so much for the help, @zeh ! 🎉

**Comment by zeh** [comment]
Nice! Thank _you_ for the report and the repro!

**Comment by zeh** [comment]
Closed as this is fixed in [v5.0.0](https://github.com/zeh/prando/releases/tag/v5.0.0).
